README.md

```diff
@@ -118,6 +123,16 @@ using following access ports:
 | IPv6                                                | 7200        |
 | IPv4, using TLS/SSL                                 | 7300        |
 
+### Code formatting
+
+To have a common look-and-feel [clang-format](https://clang.llvm.org/docs/ClangFormat.html)
+is used for code formatting. The formatting rules can be applied to the
+source code by running following make target:
+
```

Comment:
Very nice! Perhaps add "Run the formatter before committing when you're contributing to this project."?

Maybe move this to Contributing section or separate markdown file. (See other comment.)

Reply:
Will do, will split this PR in 2, one for format and one with sanitizers+memleakfix